lib/schema_generator/publisher_links_schema_generator.rb

```diff
@@ -24,7 +24,8 @@ def generate
     def properties
       {
         "links" => links,
-        "previous_version" => { "type" => "string" }
+        "previous_version" => { "type" => "string" },
+        "bulk_publishing" => { "type" => ["null", "boolean"] }
```

Comment:
Could this just be boolean? Is there a use case in passing in this value, but it being null?

Reply:
I think you're right, not sure how I came to the null or boolean conclusion.